swap_meet/vendor.py

```diff
@@ -1,2 +1,123 @@
 class Vendor:
-    pass
\ No newline at end of file
+    
+    def __init__(self, inventory=None):
+        self.inventory = [] if inventory is None else inventory
+    
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+        self.inventory.remove(item)
```

Comment:
I like how you noticed you didn't need an `else` here and as a result this reads nicely!